peripherals/usart/lpuart_v2A.yaml

```diff
@@ -0,0 +1,12 @@
+# LPUART1 found on L0 and L4 families
+
+LPUART1:
+  _include:
+    - ../peripherals/usart/_v2_ABC_common.yaml
+    - ../peripherals/usart/_v2_AB_common.yaml
+    - ../peripherals/usart/_v2_with_7bit_data.yaml
```

Comment:
all three should be `./_v2_...yaml` (no `../peripherals/usart`, paths are relative to the current file)